iidm-import-export-api/src/main/java/eu/itesla_project/iidm/datasource/Bzip2FileDataSource.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2016, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package eu.itesla_project.iidm.datasource;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.Objects;
+import org.jboss.shrinkwrap.impl.base.io.tar.bzip.BZip2CompressorInputStream;
+import org.jboss.shrinkwrap.impl.base.io.tar.bzip.BZip2CompressorOutputStream;
```

Comment:
The [shrinkwrap guidelines webpage](http://arquillian.org/guides/shrinkwrap_introduction/) encourage projects to only use the implementation packages at runtime.
> Only the API should be available upon your compilation ClassPath, while the SPI and the Implementation modules are both required for the runtime. This is to enforce good separation between classes intended for direct use and the project’s internals."

Please consider to use Apache commons-compress instead of Shrinkwrap internal BZip2InputStreamCompressor / BZip2OutputStreamCompressor. This library supports also 7zip and XZ format.